charts/testkube-operator/values.yaml

```diff
@@ -29,6 +29,47 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+webhook:
+  enabled: true
+  name: testkube-operator-webhook-admission
+  labels: {}
+  annotations: {}
+
+  patch:
+    enabled: true
+    image:
+      repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+      tag: v1.3.0
+      pullPolicy: IfNotPresent
```

Comment:
we used Always before

Reply:
so, revert to Always everywhere?

Reply:
probably, yes, for now. but, we need to create an issue (tech-debt) to finally decide about default policy